r/vignettes/developing.Rmd

```diff
@@ -101,37 +114,48 @@ brew install cmake openssl
 sudo apt install -y cmake libcurl4-openssl-dev libssl-dev
 ```
 
-### Configure the Arrow build {.tabset}
+#### Windows
+
+Currently, the R package cannot be made to work with a locally-built Arrow C++ library. This will be resolved in a future release.
+
+### Step 2 - Configure the Arrow build
```

Comment:
Same note about the `{.tabset}` here as well. If we do want to get rid of them completely, that's totally fine, but if we do get rid of the tabs, I think we should pick either user directory install or system install and only document one of those. Having both of these sections run one after the other on the the pkgdown site (which I assume is the place a large portion of people reading this read it from) is more confusing than it is helpful. Hiding one or the other with tabs makes it less distracting to document both cause one can totally ignore one/the other.